packages/next/src/server/initialize-require-hook.ts

```diff
@@ -1,12 +1 @@
-import {
-  loadRequireHook,
-  overrideBuiltInReactPackages,
-} from '../build/webpack/require-hook'
-
-loadRequireHook()
-
-const isPrebundled = false
```

Comment:
Looks like we can remove this as well now https://github.com/vercel/next.js/blob/0ceae76bf405e7268e7a7350720b7ce3ccc181ed/packages/next/src/build/index.ts#L297